conf/helmfile.d/0500.elasticsearch.yaml

```diff
@@ -193,7 +193,7 @@ releases:
           #   memory: "128Mi"
         resources:
           limits:
-            cpu: "1"
+            # cpu: "1"
```

Comment:
I removed the resource limit because we're scheduling the elasticsearch-data-* pods onto their own nodes now, so there's no reason not to let them have access to all the resources on the node. I've included a comment to this effect now.